scripts/qesap/lib/cmds.py

```diff
@@ -281,6 +281,16 @@ def ansible_command_sequence(configure_data_ansible, base_project, sequence, ver
     # 4. Start composing and accumulating the list of all needed commands
     ansible_cmd = []
     ansible_cmd_seq = []
+
+    if junit and not os.path.isdir(junit):
```

Comment:
I would write like this to be more readable: 
`junit_specified = bool(junit)
junit_is_not_dir = not path.isdir(junit)

if junit_specified and junit_is_not_dir:`

Reply:
For me it looks good but as you said it's a preference